businessScrapeMain.py

```diff
@@ -43,8 +42,7 @@ def format_uni_to_string(bus):
     Takes ASCII business name
     Returns String business name
     """
-    bus = bus.encode("utf-8")
-    return bus
+    return bus.encode("utf-8")
 
 
 # Main Yelp URL
```

Comment:
Commenting on the rest of the file below (left as an exercise because I'm lazy), see [this comment](https://github.com/BlueSkies7/vectorMath/pull/1/files#r165800579) and [this one](https://github.com/BlueSkies7/vectorMath/pull/1/files#r165801601) about how to structure your code and [this comment](https://github.com/BlueSkies7/vectorMath/pull/1/files#r165802606) about the `print` at the bottom of the file.

Reply:
Thank you! I'll take a look and clean this code up a bit 🙂